mbta/arrow
# feat: disruption validations

#### Summary of changes
**Asana Ticket:** [🏹 Arrow has validation for disruptions](https://app.asana.com/0/584764604969369/1173526948047133/f)

Validations added:
* start date shouldn't be after end date
* days of week should fall between start and end date
* any exception dates should fall between start and end date
* exception dates should be unique, and should be applicable to days of week 
* start time happens before end time 

<img width="1136" alt="Screen Shot 2020-06-02 at 11 05 06 PM" src="https://user-images.githubusercontent.com/18427346/83591304-a511c680-a525-11ea-9262-c6d900729784.png">


#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes.
